.travis.yml

```diff
@@ -8,10 +12,25 @@ php:
   - 7.0
   - hhvm
 
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
 matrix:
-  allow_failures:
-    - php: 7.0
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev SYMFONY_DEPRECATIONS_HELPER=strict
+    - php: 5.6
+      env: SYMFONY_VERSION=3.0.*@dev SYMFONY_DEPRECATIONS_HELPER=strict DEPENDENCIES=dev COMPOSER_FLAGS="--prefer-stable"
```

Comment:
why are you putting ``--prefer-stable`` here, when testing ``DEPENDENCIES=dev`` ? It defeats the testing against dev dependencies, as it will still use stable deps